pages/code_schools.js

```diff
@@ -48,6 +52,10 @@ export default class CodeSchools extends React.Component {
     this.setState({ allSchools: data, filteredSchools: data, moocSchools });
   }
 
+  handleModalOpen = locationsModalInfo => this.setState({ locationsModalInfo });
```

Comment:
What if you destructure the object here so it's crystal clear what the modal cares about it at every level of implementation?